### ISSUE TITLE
Video Generation model

### ISSUE THREAD

**Issue by wlsdml1114** [HVision-NKU/StoryDiffusion#1]
What a fantastic great work!

When will the Video Generation model be released?

**Comment by zhoudaquan** [comment]
> What a fantastic great work!
> 
> When will the Video Generation model be released?

Hi,

Thanks for your interest! For the video-related codes, we need to go through the company's legal check and wait for approval....

**Comment by wlsdml1114** [comment]
Thank you for answer!

I hope to hear a good decision. :)

**Comment by NikoBele1** [comment]
> we need to go through the company's legal check

Is it a matter of training data used or regarding the code for the video generation?

**Comment by olliacc** [comment]
> Thanks for your interest! For the video-related codes, we need to go through the company's legal check and wait for approval....

so no?

**Comment by william-P-abel** [comment]
> > Thanks for your interest! For the video-related codes, we need to go through the company's legal check and wait for approval....
> 
> so no?
ELLA SDXl got blocked, there is no way it's making it through.

**Comment by zhoudaquan** [comment]
> > we need to go through the company's legal check
> 
> Is it a matter of training data used or regarding the code for the video generation?

Hi,

thanks for your interest! The main issue is regarding the data we have used for model training.

**Comment by fakerybakery** [comment]
Hi @zhoudaquan,
Thank you for releasing StoryDiffusion! If the main issue is the training data, is there any possibility that the code could be released first, so if people want to train models independently they can do so?
Thank you!

**Comment by eliohead** [comment]
long live the open source

**Comment by Devil-Mix** [comment]
Any updates?

**Comment by Falkonar** [comment]
About 9 month I waiting somebody to release self attention open source model. Maybe some day that will happened. SORA making the same and training they own models but maybe with longer token limits.

**Comment by Devil-Mix** [comment]
We should probably set up the whole thing by our selves on Comfy..

**Comment by Falkonar** [comment]
> We should probably set up the whole thing by our selves on Comfy..

I just learning but will try to use this new AGI ) https://github.com/Doriandarko/maestro

**Comment by akk-123** [comment]
@zhoudaquan when will release video code?

**Comment by agi-dude** [comment]
No, I suppose then. Guess we are not gonna get a good open-source video generation model anytime soon.

**Comment by bmwas** [comment]
@zhoudaquan  Any idea if you will be able to release training/finetuning code on this model? Thank you.
